doc/classes/@GlobalScope.xml

```diff
@@ -2035,6 +2035,18 @@
 		<constant name="KEY_LAUNCHF" value="4194415" enum="Key">
 			Launch Shortcut F key.
 		</constant>
+		<constant name="KEY_GLOBE" value="4194416" enum="Key">
+			"Globe" key on Mac / iPad keyboard.
+		</constant>
+		<constant name="KEY_KEYBOARD" value="4194417" enum="Key">
+			"On-screen keyboard" key iPad keyboard.
```

Comment:
Preexisting typo that would be worth fixing here too:
```suggestion
			"On-screen keyboard" key on iPad keyboard.
```

Reply:
Just fixed. 

In `core/os/keyboard.cpp` some names are different from the original enum or even have spaces. Do you think I can fix it now or is it better to make a proposal?

Examples: `{Key::KP_2, "Kp 2"},`, `{Key::KEY_DELETE, "Delete"},` or `{Key::KEYBOARD, "On-screen keyboard"},`.

Reply:
> In core/os/keyboard.cpp some names are different from the original enum or even have spaces. Do you think I can fix it now or is it better to make a proposal?

That will be a breaking change since some code and data serialization might rely on the specific name format, so definitely should not be done in this PR.